src/app/components/main-menu/main-menu.module.css

```
@@ -1,7 +1,97 @@
-/*
- * Replace this with your own classes
- *
- * e.g.
- * .container {
- * }
-*/
+.placeholder {
+  width: 200px;
+  height: 100vh;
+}
+
+.container {
+  margin: 0;
+  padding: 0;
+  position: fixed;
+  width: 200px;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  background-color: bisque;
+}
+
+.logo {
+  margin-top: 25px;
+  margin-left: 25px;
+  border-radius: 350px;
+  height: 150px;
+  width: 150px;
+  cursor: pointer;
+}
+
+.test {
+  border-color: blueviolet;
+  font-size: large;
+}
+.menu {
+  position: relative;
+  /* margin: 100px auto; */
```

Remove or add TODO, for example:

```suggestion
  /* TODO: Uncomment on design phase */
  /* margin: 100px auto; */
```